packages/tooltip/test/tooltip.test.js

```diff
@@ -480,54 +480,42 @@ describe('vaadin-tooltip', () => {
       await nextFrame();
     });
 
-    describe('target is partially visible', () => {
-      beforeEach(async () => {
-        container.scrollTop = 220;
-        await waitForIntersectionObserver();
-      });
-
-      it('should open overlay when focused target is fully visible and hide otherwise', async () => {
-        tabKeyDown(target);
-        target.focus();
-        expect(overlay.opened).to.be.not.ok;
-
-        target.scrollIntoView({ block: 'center' });
-        await waitForIntersectionObserver();
-        expect(overlay.opened).to.be.true;
-
-        container.scrollTop = 0;
-        await waitForIntersectionObserver();
-        expect(overlay.opened).to.be.false;
-      });
+    beforeEach(async () => {
+      container.scrollTop = 220;
+      await waitForIntersectionObserver();
+    });
 
-      it('should open overlay when hovered target is fully visible and hide otherwise', async () => {
-        mouseenter(target);
-        expect(overlay.opened).to.be.not.ok;
+    it('should open overlay when focused target is partially visible and hide otherwise', async () => {
+      // Partially visible
+      tabKeyDown(target);
+      target.focus();
+      expect(overlay.opened).to.be.true;
 
-        target.scrollIntoView({ block: 'center' });
-        await waitForIntersectionObserver();
-        expect(overlay.opened).to.be.true;
+      // Fully visible
+      target.scrollIntoView({ block: 'center' });
+      await waitForIntersectionObserver();
+      expect(overlay.opened).to.be.true;
 
-        container.scrollTop = 0;
-        await waitForIntersectionObserver();
-        expect(overlay.opened).to.be.false;
-      });
+      // Fully hidden
+      container.scrollTop = 0;
+      await waitForIntersectionObserver();
+      expect(overlay.opened).to.be.false;
     });
 
-    describe('target is fully visible', () => {
```

Comment:
Removed this suite / test, as the test case should already be covered by the two previous tests.